tests/test_exceptions.py

```diff
@@ -300,6 +300,7 @@ def test_error_already_set_what_with_happy_exceptions(
     assert what == expected_what
 
 
+@pytest.mark.skipif("env.PYPY", reason="PyErr_NormalizeException Segmentation fault")
```

Comment:
Does this still segfault at the original Normalize location?

Reply:
Yes: https://github.com/pybind/pybind11/actions/runs/2386632518